images/custom-error-pages/rootfs/www/403-refresh-fresha.html

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
shouldn't this be json file instead of html?

Reply:
fixed